### PR TITLE
Feat: add connext, stargate, swapgeneric bridge 

### PR DESCRIPTION
**Pull request by xdev1016**

### CODE REVIEW

**Review by michaelnguyen102 — CHANGES_REQUESTED** [review]

File: bridge/contracts/HecBridgeSplitter.sol [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/utils/math/SafeMathUpgradeable.sol";
+import {IHecBridgeSplitterInterface} from "./interface/IHecBridgeSplitterInterface.sol";
+import {ILiFi} from "./interface/Libraries/ILiFi.sol";
+import {LibSwap} from "./interface/Libraries/LibSwap.sol";
+
+/**
+ * @title HecBridgeSplitter
+ */
+contract HecBridgeSplitter is
```
Comment (michaelnguyen102) [inline bridge/contracts/HecBridgeSplitter.sol:17]: @xiangdev1016  I am reviewing your PR and noticed something about your contract design.

Pros: your HecBridgeSplitter contract would work but your design poses several risks:
- You identified earlier: Li.Fi can remove/add existing bridges at their own discretion.
- What if later on Stargate or NXTP change their contract and that would jeopardize our bridge contract immediately. 

Recommendation:
- HecBridgeSplitter should not contain the implementation of Starbridge or NXTP. Instead, we should use the Dependency Injection Design Pattern to refactor the bridge operation from Starbridge or any bridge service to its own contract. 
- In other words, here's specifically what I would do:
  - Refactor StarBridge and NXTP to its separate smart contract
  - Using the iterator pattern, HecBridgeSplitter would keep a list of bridges like Stargate, etc. Admin can also add/update/remove bridges from HecBridgeSplitter contract.
  - The dApp simply passes the bridge contract address to activate that bridge

**Review by michaelnguyen102 — CHANGES_REQUESTED** [review]
